integration/test-integration/src/test/resources/dockerCompose/backend-service.yaml

```diff
@@ -1,6 +1,6 @@
       - "5006:5006"
   mockBackend:
-    image: wso2/choreo-connect-mock-backend:latest
+    image: wso2/choreo-connect-mock-backend:1.0.0-beta-SNAPSHOT
```

Comment:
Do we need to version the mock service?
If so shall we add it here to get the version replaced with version updates?
https://github.com/wso2/product-microgateway/blob/8c5b7c1421a2568c9ef19575ac6dcbc1123740c0/pom.xml#L150-L164

Reply:
Already have :) . It is here https://github.com/wso2/product-microgateway/pull/2399/files#diff-0674bcf8c92221bb6df454ef1f134c65bfe2b6c18d95041b3590b1be62c7f6e5R76